### DISCUSSION TITLE
How to merge two refs in react-select components

### DISCUSSION THREAD

**Discussion by ImthiyasMd** [react-hook-form/react-hook-form#8816] [Q&A]
Hello, Could you tell me how to combined controller render ref prop and normal ref prop and pass it to the react-select component ref to access both?

https://codesandbox.io/s/react-select-with-react-hook-form-forked-uq5ysp?file=/src/App.js

Thanks in Advance!

**Comment by Moshyfawn** [comment]
https://react-hook-form.com/faqs/#question5

**Comment by bryanprimus** [comment]
https://react-hook-form.com/faqs#Howtosharerefusage
